Monitoring in a Box/Ansible/host_vars/mib1.yaml

```diff
@@ -0,0 +1,30 @@
+### VM variables
+## Which image format to use
+# qcow2
+# vmdk
+# raw
+ImageFormat: vmdk
+
+## New VM ID number
+NewVmID: "240"
+
+## New VM name
+NewVmName: "MiB-Host"
+
+## New account username
+NewVmUsername: mib
+
+## VM's IP address
+NewVmIP: 10.0.1.240
+
+# Pve node sudo password
+ansible_become_pass: [key in pve node sudo password]
```

Comment:
Sorry can you remind me what this is? I believe the `--ask-become-pass` flag will provide the required password.

Reply:
This is to specify the become password for our pve node. Which will also be set as the mib vm password during the set up phrase

Reply:
In this case, omit the variable because putting the password here is an operational risk. Raw passwords should never be included in the Ansible script if avoidable. If you commit the code with the password, you'll have to reset the password on all affected nodes.

The more sound way is to specify `--ask-become-pass`.

Reply:
Alright I will remove the `ansible_become_pass` variable from the script.